### ISSUE TITLE
No left stick click functionality

### ISSUE THREAD

**Issue by AWildBeard** [kozec/sc-controller#192]
So, I've googled around and have had no results. I have bound the left stick click to allot of keys and none of them have worked. I know the left stick click on the controller is not broke because Steam receives the left stick click just fine without problems when I disable scc-daemon.

**Comment by kozec** [comment]
Weird, I'm clicking with stick right now. Can you try enabling input test mode and check if stick is blinking in UI? It's menu->settings-> tab advanced ->1st option.

**Comment by AWildBeard** [comment]
So, I had sc-controller on a startup program (i3) that would launch sc-controller on a different workspace and steam on a 30 second delay after so it doesn't interfere (same workspace). As a test, I disabled both on startup and manually launched them; sc-controller first (waited until the light on the top left was green) then steam immediately after. So, my solution now is to just manually launch the two (not terrible since I don't always need the two to start if I'm doing work). I did enable the settings that you suggested, and will watch to see if the problem reproduces. Going to close since my particular problem is no longer an issue (but probably still present) @kozec
